src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/DetailControl.xaml.cs

```diff
@@ -165,6 +165,7 @@ private void ExecuteUserAction(UserAction action, NuGetActionType actionType)
                     nugetUi.RecommendedCount = model.RecommendedCount;
                     nugetUi.RecommendPackages = model.RecommendPackages;
                     nugetUi.RecommenderVersion = model.RecommenderVersion;
+                    nugetUi.VulnerablePackagesCount = model.IsPackageVulnerable? 1: 0;
```

Comment:
```Suggestion
                    nugetUi.VulnerablePackagesCount = model.IsPackageVulnerable ? 1: 0;
```

Reply:
Is this actually the vulnerable package count or just a 1 or 0 determining if there are any vulnerable packages?

Reply:
Is just 1 or 0 indicating if there are vulnerable package to install, update or uninstall

Reply:
Change of plans. It indicates the number of top-level vulnerable packages found to install, update or uninstall.

Reply:
Should make it explicit that we're talking about top level vulnerable packages in the event name?
I imagine transitive ones becomes interesting eventually.

Reply:
Fixed.